src/fidesops/ops/schemas/saas/strategy_configuration.py

```diff
@@ -87,6 +87,27 @@ class CursorPaginationConfiguration(StrategyConfiguration):
     field: str
 
 
+class ApiKeyAuthenticationConfiguration(StrategyConfiguration):
+    """
+    API key parameter to be added in as a header or query param
+    """
+
+    headers: Optional[List[QueryParam]]
```

Comment:
Minor, but we have a `Header` type in `saas_config.py`

Reply:
oh nice. let me go ahead and change that, feels quite a bit cleaner.